.github/linters/.yaml-lint.yml

```
@@ -0,0 +1,34 @@
+extends: default
+
+rules:
+  document-start: disable
+
+  indentation:
+    spaces: 2
+    level: error # Enforce strict indentation
```

yamllint

[warning] too few spaces before comment

(comments)